ci-operator/jobs/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main-presubmits.yaml

```diff
@@ -62,6 +62,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )dataplane-operator-build-deploy,?($|\s.*)
+  - agent: kubernetes
```

Comment:
The way it's configured, both `build-deploy` and `build-deploy-kuttl` will run for every PR of dataplane-operator and duplicate the build and deploy steps. It would be a good idea to set `always_run: false` in the `build-deploy` job (line 4 of this file)

Reply:
Thanks. I have make this improvement in https://github.com/openshift/release/pull/39513